### PR TITLE
New Field awards in Content and Person

### PR DESCRIPTION
**Pull request by ljeyaratnam**

### CODE REVIEW

**Comment by Spartakovic** [comment]
tests?

File: src/main/java/org/atlasapi/persistence/media/entity/AwardTranslator.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Iterables;
+import com.metabroadcast.common.persistence.translator.TranslatorUtils;
+import com.mongodb.BasicDBList;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+
+public class AwardTranslator {
+
+    public static final String OUT_COME = "outcome";
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/persistence/media/entity/AwardTranslator.java]: Outcome is a single word. There shouldn't be an underscore in the var name

File: src/main/java/org/atlasapi/persistence/media/entity/ContentTranslator.java [hunk 2]
```diff
@@ -236,6 +242,7 @@ public DBObject toDBObject(DBObject dbObject, Content entity) {
             }
             dbObject.put(VERSIONS_KEY, list);
         }
+        dbObject.put(AWARDS, awardTranslator.toDBList(entity.getAwards()));
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/persistence/media/entity/ContentTranslator.java]: Can entity.getAwards() be null? If so the AwardTranslator could NPE

File: src/test/java/org/atlasapi/persistence/media/entity/AwardTranslatorTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package org.atlasapi.persistence.media.entity;
+
+import static junit.framework.TestCase.assertEquals;
+
+import org.atlasapi.media.entity.Award;
+import org.junit.Test;
+
+import com.mongodb.DBObject;
+
+public class AwardTranslatorTest {
```
Comment (Dalamar42) [inline src/test/java/org/atlasapi/persistence/media/entity/AwardTranslatorTest.java]: This test is only covering serialisation into a DBObject. It does not cover deserialisation. You should do something like this:

```
DBObject serialised = translator.toDBObject(award);
Award deserialised = translator.fromDBObject(serialised);
```

And then check for equality of every field between `award` and `deserialised`

File: src/main/java/org/atlasapi/persistence/media/entity/ContentTranslator.java (hunk 2 above)
Comment (Dalamar42) [inline src/main/java/org/atlasapi/persistence/media/entity/ContentTranslator.java]: Never mind. I saw it's always going to be at least the empty set

**Comment by Dalamar42** [comment]
LGTM

**Pull request merged**